README.md

```diff
@@ -53,9 +53,16 @@ To create deep copies, set the `UseDeepCloning` property on the `MapperAttribute
 
 On each mapping method declaration property mappings can be customized.
 If a property on the target has a different name than on the source, the `MapPropertyAttribute` can be applied.
-Flattening is not yet supported.
 If a property should be ignored, the `MapperIgnoreAttribute` can be used.
 
+##### Flattening and unflattening
```

Comment:
The hierarchy of the titles seems odd. Currently, "Properties" and "Flattening and unflattening" are both children of the "Copy behaviour" title... IMO they should all use the same header hierarchy

Reply:
Adjusted.